easybuild/easyblocks/g/gcc.py

```diff
@@ -689,6 +779,20 @@ def build_step(self):
         super(EB_GCC, self).build_step()
 
     # make install is just standard install_step, nothing special there
+    def install_step(self, *args, **kwargs):
+        # When building AMD offloading do not install the LLVM source files with 'make install'
```

Comment:
This comment should be expanded a bit, it's more than not using `make install`, we're doing something else instead?